src/scrape/scrape_job_test.go

```diff
@@ -200,6 +233,40 @@ func TestPopulateK8s(t *testing.T) {
 	}
 }
 
+func TestRestartCountDeltaValues(t *testing.T) {
+	intgr, err := integration.New("test", "test", integration.InMemoryStore())
```

Comment:
can we add t.parallel()?

Reply:
If you use `t.Parallel` you'll need to wrap this with a mutex to avoid a race in the SDK. It's not particularly good, but the problem is already solved :)
https://github.com/newrelic/nri-kubernetes/blob/main/internal/testutil/integration.go#L18

Reply:
Added here: [bc62e84](https://github.com/newrelic/nri-kubernetes/pull/382/commits/bc62e845386de80d99d1d5b928fc25d5b72ecd98)